ratis-shell/pom.xml

```diff
@@ -48,6 +48,14 @@
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-simple</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.ratis</groupId>
+      <artifactId>ratis-grpc</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.ratis</groupId>
+      <artifactId>ratis-netty</artifactId>
+    </dependency>
```

Comment:
@DaveTeng0 , Thanks  a lot for working on this!  This refactoring change should not add new dependencies.  I understand that the new dependencies is for supporting security.  Please make the security in a separated JIRE.

Reply:
sure! removed from this PR.